dgraphtest/local_cluster.go

```diff
@@ -781,7 +781,17 @@ func (c *LocalCluster) checkDgraphVersion(containerID string) error {
 	if err != nil {
 		return errors.Wrapf(err, "error during checkDgraphVersion for container [%v]", containerID)
 	}
-	if !strings.Contains(contLogs, fmt.Sprintf("Dgraph version   : %v", c.GetVersion())) {
+	index := strings.Index(contLogs, "Dgraph version   : ")
+	running := strings.Fields(contLogs[index : index+70])[3] // 70 is arbitrary
```

Comment:
yes, because we do health check on the container before checking for the version of Dgraph. The logs that are printed are a lot more than 70 characters by the time the health check passes.

Reply:
Ok. Since this is a test driver code, I'm not being too picky about it.

Reply:
yeah, we could probably be do something smarter, but we don't need to do it. This is simple and works really well.